.vitepress/local.js.dist

```diff
@@ -53,3 +53,22 @@ export const man_paths = []
 //
 // Default: [ 'docs/core/plugins/*.md' ]
 export const plugin_paths = []
+
+// Enable additional labels to support in Dovecot-specific markdown
+// processing (i.e. [[xyz,...]]).
+export const markdown_extend = {
+
+	// Init function. Return value is configuration options to add to markdown
+	// object.
+	// init: () => { return {} },
+
+	// Opening tag function. Returns opening tag.
+	// open: (mode, parts, opts, env) => { return '' },
+
+	// Body function. Returns body text.
+	// body: (mode, env) => { return '' },
+
+	// Close tag function. Returns closing tag.
+	// close: (mode, env) => { return '' },
+
+}
```

Comment:
Huh, makes you wonder, why we never used TypeScript.
(To be interpreted as a comment, not as an action item.)

Reply:
See number 2 above.  We need to be doing just enough to enable us to be able to maintain the code and build good documentation.  We are not shipping any of this code, so asking developers to not only learn basic Javascript but TypeScript on top of it seems to be asking too much. (Security isn't a big issue here because we are just doing enough javascript to statically generate content; this isn't the use-case where active user data is being processed and sent across the Interwebs, for example.)

Reply:
OK, I see now.